interview-questions.md

```diff
@@ -7,44 +8,54 @@ Then, research the question to expand on your answer. Even if you feel you have
 
```

Comment:
And that is exactly how you do interview questions. I can truly see that you understand the concepts and are willing to not only do research but also form your own understanding. Truly appreciate your thorough approach.